INSTALL.rst

```diff
@@ -0,0 +1,363 @@
+.. The source of this document is INSTALL. During the doc build process,
+.. this file is copied over to doc/users/installing.rst.
+.. Therefore, you must edit INSTALL, *not* doc/users/installing.rst!
+.. _pip: https://pypi.python.org/pypi/pip/
+
+==========
+Installing
+==========
+
+.. note::
+
+    If you wish to contribute to the project, it's recommended you
+    :ref:`install the latest development version<install_from_source>`.
+
+
+
+.. contents.. 
+
+Installing an official release
+==============================
+
+Matplotlib and most of its dependencies all available as wheel packages for
+OSX, windows and many linux distributions::
+
+  pip install -U matplotlib
+
+Third-party distributions of Matplotlib
+=======================================
+
+Scientific Python distributions: Conda, Canopy...
+--------------------------------------------------
+
+The first option is to use one of the pre-packaged Python distributions that
+already provide Matplotlib built-in.  Both `Anaconda
+<https://www.continuum.io/downloads/>`_ and `Canopy
+<https://www.enthought.com/products/canopy/>`_ are both excellent choices that
+"just work" out of the box for Windows, OSX and common Linux platforms.  Both
+of these distributions include Matplotlib and *lots* of other useful tools.
+
+Linux : using your package manager
+----------------------------------
+
+If you are on Linux, you might prefer to use your package manager.  Matplotlib
+is packaged for almost every major Linux distribution.
+
+* Debian / Ubuntu : ``sudo apt-get install python-matplotlib``
+* Fedora / Redhat : ``sudo yum install python-matplotlib``
+
+.. _installing_windows:
+
+Windows
+-------
+
+We strongly recommend using `scipy-stack compatible Python distributions
+<http://www.scipy.org/install.html>`_ such as WinPython, Python(x,y),
+Enthought Canopy, or Continuum Anaconda, which have Matplotlib and many of its
+dependencies, plus other useful packages, preinstalled.
+
+For `standard Python <https://www.python.org/downloads/>`_ installations,
+install Matplotlib using pip_::
+
+    python -m pip install -U pip setuptools
+    python -m pip install matplotlib
+
+In case Python 2.7 or 3.4 are not installed for all users,
+the Microsoft Visual C++ 2008
+(`64 bit <https://www.microsoft.com/en-us/download/details.aspx?id=15336>`__
+or
+`32 bit <https://www.microsoft.com/en-us/download/details.aspx?id=29>`__
+for Python 2.7) or Microsoft Visual C++ 2010
+(`64 bit <https://www.microsoft.com/en-us/download/details.aspx?id=14632>`__
+or
+`32 bit <https://www.microsoft.com/en-us/download/details.aspx?id=5555>`__
+for Python 3.4) redistributable packages need to be installed.
```

Comment:
No clue: this was not written by me, and I don't feel comfortable changing this section. I haven't touched a windows machine in 10years.